playwright-e2e/dsm/component/tables/participantListTable.ts

```diff
@@ -40,6 +59,11 @@ export class ParticipantListTable extends Table {
     return this.page.locator(`//table/tbody/tr`).nth(position);
   }
 
+  /* Locators */
+  public get rowsCount(): Promise<number> {
```

Comment:
I think `rowsCount` and `rowsXpath` func are not needed because similar func already exists in `Table`.